### DISCUSSION TITLE
New install fails to add accessories

### DISCUSSION THREAD

**Discussion by azpflyer** [dgreif/ring#782] [Q&A] [answered]
I recently built a pi-based HOOBS (v4.0.100) and installed the Ring plug-in (v9.21.1) and it couldn’t have gone smoother. Added the device in HomeKit using the QR code and my camera and doorbell showed right up. Couldn’t have been easier.

Built a second for my daughter who has a ring base station with one doorbell and 26 alarm sensors. HOOBS on the box is v4.0.109 and Ring plug-in is 9.21.2. Ring account linked fine, scanned the code and the bridge added to HomeKit, but none of the accessories or camera show up. Uninstalled the plugin from HOOBS and HomeKit and started over…no joy.

The NEST plug-in installed and all the device (accessories) showed up in HomeKit just fine.

Looking at the log file, the ring devices all show up fine:

9/30/2021, 7:40:30 PMRing BridgeBridge is running on port 51826.
9/30/2021, 7:40:30 PMRing BridgeRingFound the following locations:
9/30/2021, 7:40:30 PMRing BridgeRing  locationId: <nnnnnn-Cityname>
9/30/2021, 7:40:30 PMRing BridgeRing  locationId: <nnnnnn-location name>
9/30/2021, 7:40:30 PMRing BridgeRingCreating location socket.io connection - City
9/30/2021, 7:40:30 PMRing BridgeRingCreating location socket.io connection - location
9/30/2021, 7:40:30 PMRing BridgeRingRing connected to socket.io server
9/30/2021, 7:40:30 PMRing BridgeRingERRORRing base_station_v1 nnnnnnnn is offline or on cellular backup.  Waiting for status to change
9/30/2021, 7:40:30 PMRing BridgeRingRing connected to socket.io server
9/30/2021, 7:40:31 PMRing BridgeRingConfiguring 1 cameras and 26 devices for location xxxxxx
9/30/2021, 7:40:31 PMRing BridgeRingAdding new accessory xxxxxxx hub.redsky Base Station
9/30/2021, 7:40:31 PMRing BridgeRingAdding new accessory xxxxxxx security-panel Alarm
9/30/2021, 7:40:31 PMRing BridgeRingAdding new accessory xxxxxxx sensor.contact Balcony Door
.
.
.

All accessories seem to add successfully. The only anomaly seems to be the base not being connected to an alarm provider, which I wouldn’t think would affect the accessories not showing in HomeKit.

is there possibly a version incompatibility between HOOBS and the plug-in?

Thanks in advance!

**Comment by dgreif** [comment, marked as answer]
The plugin is unable to fetch devices while an alarm base station is offline or on cellular backup. This is intentional, because we don't want to accidentally remove devices from that base station if you happen to restart while it's offline.  If you get it plugged in and connected to the internet, the accessories should be found and get added to HomeKit.

**Comment by azpflyer** [comment]
Thanks!!
